dir.props

```diff
@@ -559,6 +580,9 @@
     <PackagesBasePath Condition="'$(PackagesBasePath)'==''">$(BinDir)$(OSPlatformConfig)</PackagesBasePath>
     <PackageOutputPath Condition="'$(PackageOutputPath)'==''">$(PackageOutputRoot)$(ConfigurationGroup)/</PackageOutputPath>
     <SymbolPackageOutputPath Condition="'$(SymbolPackageOutputPath)'==''">$(PackageOutputPath)symbols/</SymbolPackageOutputPath>
+  
+    <!-- When testing against packages, we want to run against just the localpackages cache -->
+    <PackagesDir Condition="'$(BuildTestsAgainstPackages)'=='true'">$(PackageInstallPath)</PackagesDir>
```

Comment:
Have we verified that nothing in the buildtools logic (or other parts of the CoreFX build logic) depends on `$(PackagesDir)` being corefx\packages?  I searched for usages of this variable in buildtools, and one example that I'm wondering about is https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets#L35.  Will the `$(PackageInstallPath)` directory contain xunit.runner.console?  I don't see it in buildagainstpackagesdeps.props.  Ditto for microsoft.xunit.runner.uwp, which is used on L45 of publishtest.targets.

Another example is at https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets#L37; will OpenCover be in `$(PackageInstallPath)`?